### ISSUE TITLE
runLocally overrides hostname

### ISSUE THREAD

**Issue by rvanlaak** [deployphp/deployer#1983]
Because runLocally passes `localhost` to the process as it's hostname, using `{{hostname}}` in processes you want to run locally is not possible.

## Expected

given that hostname is dev.example.com

```php
runLocally('ssh-keyscan -H {{hostname}} >> ~/.ssh/known_hosts');
# or
run('ssh-keyscan -H {{hostname}} >> ~/.ssh/known_hosts')->local();
```

should run

```sh
ssh-keyscan -H dev.example.com >> ~/.ssh/known_hosts
```

## Actual result

```sh
ssh-keyscan -H localhost >> ~/.ssh/known_hosts
```

## Workaround

Not using `runLocally` or `->local()`:

```php
exec(parse('ssh-keyscan -H {{hostname}} >> ~/.ssh/known_hosts'));
```

or parsing the script before running it locally

```php
runLocally(parse('ssh-keyscan -H {{hostname}} >> ~/.ssh/known_hosts'));
```

**Comment by xuanquynh** [comment]
Which deployer version do you use?

**Comment by rvanlaak** [comment]
We used https://hub.docker.com/r/mwienk/php-deployer which was last updated 2yrs ago. Running `docker run --rm mwienk/php-deployer dep -v` does output `master`, so probably somewhere around the release of `v6.0.x`.

After building our own image (and including `rsync`) we found out that this isn't the case with the latest tag `v6.7.3` as we were able to successfully deploy. In other words, this issue can be closed 👍 

Future confusion would be solved by adding that Dockerfile to this repo, and configuring automated builds on Docker hub. @antonmedv shall I create a PR for that?
